dashboard/app/models/census/state_course_code.rb

```diff
@@ -0,0 +1,2 @@
+class Census::StateCourseCode < ApplicationRecord
```

Comment:
I removed the Model class, since it isn't being used yet, and previous migration Pull Requests (#18136) have not included the model.  I think it will be generated with annotations once the migration is run.